SilKit/source/CMakeLists.txt

```
@@ -41,7 +41,7 @@ set(GIT_DIR "${PROJECT_SOURCE_DIR}/../.git")
 set(GIT_HEAD_FILE "${GIT_DIR}/HEAD")
 set(VERSION_MACROS_HPP ${CMAKE_CURRENT_BINARY_DIR}/version_macros.hpp)
 if(EXISTS ${CMAKE_CURRENT_LIST_DIR}/version_macros.hpp)
-    message(STATUS "SILKIT re-using deployed version_macros.hpp")
+    message(STATUS "SILKIT: re-using deployed version_macros.hpp")
```

```suggestion
    message(STATUS "SILKIT: using deployed version_macros.hpp")
```

But it's fine either way - to me this just reads better